src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/Combination/paginated_list_header.html.twig

```diff
@@ -0,0 +1,63 @@
+{#**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ *#}
+{# Combination filters for paginated list #}
+<div id="combinations_filters"
+     data-translations="{{ {
+       'filters.label': 'Filter by:'|trans({}, 'Admin.Actions'),
+       'filters.clear': 'Clear filter|Clear %filtersNb% filters'|trans({}, 'Admin.Actions'),
+     }|json_encode }}"
+></div>
+
+{#    @todo: not styled #}
+<div id="combination-actions">
+  <button id="combination-bulk-form-btn" class="btn btn-outline-primary" type="button">
+    {{ 'Apply bulk action'|trans({}, 'Admin.Actions') }}
```

Comment:
What is the action exactly? The wording should be more specific. For example "Delete all combinations" or "Duplicate all combinations"?

Reply:
This is not the final display, it's just temporary quick button for development The buttons will look like this:
![Capture d’écran 2022-01-27 à 11 14 02](https://user-images.githubusercontent.com/13801017/151338988-04ea582d-9f59-45f6-b3b0-8f53da22fb98.png)

By the way @zuk3975 you will have to integrate this drop down buttons container in this PR I think, with only the Edit part for now of course

Reply:
Ok so if I understand well the wording should be "Bulk actions" only, right?

Reply:
In the dropdown button yes, and the "Edit %s combinations" in another button